{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/authentication\/oauthprovidererror\/recoverysuggestion"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.ibm.iOS-SDK\/documentation\/Authentication\/OAuthProviderError\/recoverySuggestion","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.recoverySuggestion"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"role":"symbol","title":"recoverySuggestion","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"symbolKind":"property","externalID":"s:10Foundation14LocalizedErrorPAAE18recoverySuggestionSSSgvp::SYNTHESIZED::s:14Authentication18OAuthProviderErrorO","extendedModule":"Foundation","modules":[{"name":"Authentication"}]},"hierarchy":{"paths":[["doc:\/\/com.ibm.iOS-SDK\/documentation\/Authentication","doc:\/\/com.ibm.iOS-SDK\/documentation\/Authentication\/OAuthProviderError","doc:\/\/com.ibm.iOS-SDK\/documentation\/Authentication\/OAuthProviderError\/LocalizedError-Implementations"]]},"references":{"doc://com.ibm.iOS-SDK/documentation/Authentication/OAuthProviderError/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/com.ibm.iOS-SDK\/documentation\/Authentication\/OAuthProviderError\/LocalizedError-Implementations","kind":"article","type":"topic","url":"\/documentation\/authentication\/oauthprovidererror\/localizederror-implementations"},"doc://com.ibm.iOS-SDK/documentation/Authentication/OAuthProviderError/recoverySuggestion":{"role":"symbol","title":"recoverySuggestion","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/com.ibm.iOS-SDK\/documentation\/Authentication\/OAuthProviderError\/recoverySuggestion","kind":"symbol","type":"topic","url":"\/documentation\/authentication\/oauthprovidererror\/recoverysuggestion"},"doc://com.ibm.iOS-SDK/documentation/Authentication/OAuthProviderError":{"role":"symbol","title":"OAuthProviderError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"OAuthProviderError"}],"abstract":[{"type":"text","text":"A type that indicates when OAuth operartion encounters an error."}],"identifier":"doc:\/\/com.ibm.iOS-SDK\/documentation\/Authentication\/OAuthProviderError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"OAuthProviderError"}],"url":"\/documentation\/authentication\/oauthprovidererror"},"doc://com.ibm.iOS-SDK/documentation/Authentication":{"role":"collection","title":"Authentication","abstract":[],"identifier":"doc:\/\/com.ibm.iOS-SDK\/documentation\/Authentication","kind":"symbol","type":"topic","url":"\/documentation\/authentication"}}}